rasa/nlu/extractors/spacy_entity_extractor.py

```diff
@@ -19,7 +19,9 @@
 
 
 @DefaultV1Recipe.register(
-    DefaultV1Recipe.ComponentType.ENTITY_EXTRACTOR, is_trainable=False
+    DefaultV1Recipe.ComponentType.ENTITY_EXTRACTOR,
+    is_trainable=False,
+    model_from="SpacyNLPGraphComponent",
```

Comment:
This spacy component needs the actual model, not just the stored stuff in the messages